.ci/pipeline/release_matrix_job.yaml

```diff
@@ -0,0 +1,65 @@
+---
+job: LIBXLIO-release
+# step_allow_single_selector: false
+registry_host: harbor.mellanox.com
+registry_auth: 1daaea28-800e-425f-a91f-3bd3e9136eea
+registry_path: /swx-infra/media/libxlio
+
+kubernetes:
+  privileged: true
+  cloud: il-ipp-blossom-prod
+  nodeSelector: 'beta.kubernetes.io/os=linux'
+  namespace: swx-media
+  limits: '{memory: 8Gi, cpu: 7000m}'
+  requests: '{memory: 8Gi, cpu: 7000m}'
+
+env:
+  MAIL_FROM: jenkins@nvidia.com
+
+volumes:
+  # Default release location
+  - {mountPath: /auto/sw/release/sw_acceleration, hostPath: /auto/sw/release/sw_acceleration}
+  # User profile for release
+  - {mountPath: /var/home/swx-jenkins, hostPath: /labhome/swx-jenkins}
+
+runs_on_dockers:
+  - {file: '.ci/dockerfiles/Dockerfile.rhel8.6.release', name: 'rhel8.6-release', build_args: '--no-cache', category: 'tool', arch: 'x86_64', tag: '20240718'}
+
+
+steps:
+  - name: Release
```

Comment:
i want to seperate into 2 steps (it does work on same container) 1 for dpcp build and one for xlio build